src/main/java/com/cloudant/client/api/CloudantClient.java

```diff
@@ -128,10 +127,6 @@
  * <h2>Replication</h2>
  * Replication {@link Replication account.replication()} and {@link Replicator account.replicator()}
  *
- * <h2>Shutting down</h2>
- * At the end of a client usage; it's useful to call {@link #shutdown() client.shutdown()} to
- * ensure proper release of resources.
- *

```

Comment:
This should be reverted too, even though shutdown is now a No-op. Its better to have people still call so they are used to doing it in the event we need to make a change.